### ISSUE TITLE
Default currency symbol invalid

### ISSUE THREAD

**Issue by kittsville** [kittsville/WP-Librarian#62]
On a default WP-Librarian install the currency symbol is supposed to be initilised to the pound symbol £. Instead it fails and displays &amp. This is happening, I think, because the default setting is already an HTML entity `&pound` so when it is saved as a setting the string passes through sanitisation and the `&` symbol is seen as the currency symbol and converted into it's HTML entity `&amp` which is then what appears on the default install.

**Comment by kittsville** [comment]
Ah, no, it's sillier than that. It's caused by this horrible mess I introduced a long while back:
```
htmlentities(substr(iconv('UTF-8', 'ISO-8859-15', trim($raw[0])), 0, 4), ENT_QUOTES, 'ISO-8859-15')
```
It's designed for handling input from the user but not for anything else. As a result the default value gets mangled.

**Comment by kittsville** [comment]
Character encoding conversion was unnecessary and has been removed. Default value has been changed from pre-encoded character to actual pound symbol. Fixed in 56ea045c9525dffe0cdaf7a6b0fa3616ed25ba83
